package thatline.localup.auth.exception

import thatline.localup.exception.BaseException

class InvalidCredentialsException : BaseException("INVALID_CREDENTIALS")
